docs/source/examples.rst

```diff
@@ -207,6 +207,19 @@ are strongly recommended for multi-fidelity HPO with the SageMaker backend.
 More details are found in
 `this tutorial <tutorials/benchmarking/bm_sagemaker.rst#using-sageMaker-managed-warm-pools>`__.
 
+Retrieving the best checkpoint
+=================================
+
+.. literalinclude:: ../../examples/launch_standalone_bayesian_optimization.py
+   :caption: examples/launch_standalone_bayesian_optimization.py
+   :lines: 13-
+
+Syne Tune combines a scheduler (HPO algorithm) with a backend to provide a
```

Comment:
Copy and pasted from another paragraph? Also, does not include your example

Reply:
That's a type, it is fixed.